src/Model/Order/Order.php

```diff
@@ -34,10 +41,22 @@ final class Order extends AggregateRoot implements SerialisableAggregateRoot
     private $businessAddress;
     /** @var  NonNullContactPerson $contactPerson */
     private $contactPerson;
-    /** @var  NonNullOrderIsPaid $hasPaid */
-    private $isPaid;
+    /** @var  NonNullPaymentStatus $paymentStatus */
+    private $paymentStatus;
     /** @var  NonNullOrderAmount $amount */
     private $amount;
+    /** @var  DatePaid $datePaid */
+    private $datePaid;
```

Comment:
I think that it would be good to have a payments property on the Order aggregate as it's possible for it to receive multiple payments. If more than one payment was made the first payment would be overwritten. The payments property should be an EntitySet.